api/types/types.go

```diff
@@ -352,9 +353,9 @@ type NetworkSettingsBase struct {
 	SecondaryIPv6Addresses []network.Address
 }
 
-// DefaultNetworkSettings holds network information
-// during the 2 release deprecation period.
-// It will be removed in Docker 1.11.
+// DefaultNetworkSettings holds network information for the default bridge. It was initially holding state data for the
+// network a container was connected to at a time where only a single network per container was allowed (ie. prior to
+// v1.09, see moby/moby#16645).
 type DefaultNetworkSettings struct {
```

Comment:
This might need some tweaking as well, more so because this  doc is the user-facing documentation for our API types (we can include further "hidden" comments somewhere if needed though).

"it was initially" makes it sound as if it's no longer doing so; but it's still used, correct?

(Wondering if we need the full history lesson for the documentation).

If these are the "to be removed" fields, perhaps we need to deprecate them "but still used for API < xxx responses".

```
// DefaultNetworkSettings holds network information for the default (bridge)
// network, which was the only network before Docker v1.9.0.
//
// Deprecated: ...
```


Maybe the "deprecated works better "per-field" in that case 🤔